rocAL/rocAL/include/decoders/image/open_cv_decoder.h

```diff
@@ -65,6 +65,7 @@ class CVDecoder : public Decoder {
 
     bool is_partial_decoder() override { return _is_partial_decoder; }
     void set_bbox_coords(std::vector <float> bbox_coord) override { _bbox_coord = bbox_coord;}
+    void set_crop_window(CropWindow &crop_wind) override {};
```

Comment:
Should this be set as _crop_wind = crop_wind?

Reply:
Yes, changed this to set the _crop_wind = crop_wind